rbi/stdlib/logger.rbi

```diff
@@ -429,7 +429,7 @@ class Logger
 
   sig do
     params(
-      logdev: T.any(String, IO, StringIO),
+      logdev: T.any(String, IO, NilClass),
```

Comment:
Any reason to delete `StringIO`?

Seems like it was ~recently added in https://github.com/sorbet/sorbet/pull/3106

Reply:
Ack, that was my mistake. I'll fix now.

Reply:
No problem! Thanks for being so responsive. I'm testing this at Stripe right now, but it's looking good!

Will give you an update ASAP

Reply:
Thank you for being so responsive! It's a relatively small thing, but it's helpful for my work, too!